addons/test_mail/tests/test_mail_multicompany.py

```diff
@@ -90,3 +90,17 @@ def test_notify_reply_to_computation_mc(self):
                     "%s %s" % (self.user_employee_c2.company_id.name, test_record.name),
                     "%s@%s" % (self.alias_catchall, self.alias_domain)))
             )
+
+        # Test3: get company from record (company_id field)
+        new_company = self.env['res.company'].sudo().create({'name': 'ELIT'})
```

Comment:
Isn't "_activate_multi_company" already creating other companies ?

Reply:
just wanted to show a distinction from the other ones to avoid confusion I can reuse an already existing one

Reply:
Would create at beginning of test then (company_3 probably).